tests/unit/aws/test_access.py

```diff
@@ -121,7 +122,13 @@ def test_create_external_locations(mock_ws, installation_multiple_roles, mock_aw
         ),
     ]
     aws_resource_permissions = AWSResourcePermissions(
-        installation_multiple_roles, mock_ws, backend, mock_aws, locations, "ucx"
+        installation_multiple_roles,
+        mock_ws,
+        backend,
+        mock_aws,
+        locations,
+        "ucx",
+        create_autospec(PrincipalACL),
```

Comment:
Blocker: you've added mocks for AwsACL, but you don't assert if methods from those mocks called, making these tests broken